src/HistoryQuery/HistoryQuery.class.js

```diff
@@ -98,11 +103,12 @@ class HistoryQuery {
     // }
 
     // const { api, enabled, name } = this.application
-    // this.north = enabled ? ApiFactory.create(api, this.application, this.engine) : null
+    // this.north = enabled ? thi.createNorth(api, this.application) : null
```

Comment:
I guess these comments will be removed. watch the thi instead of this.

Reply:
Yes. The commented piece of code will be reactivated when the import phase will be implemented.

Reply:
ok